### ISSUE TITLE
Not able to login in ipa tool to generate ipa file 

### ISSUE THREAD

**Issue by Tarun5139** [majd/ipatool#293]
### What happened?

Hello I am not able to login 
in ipa tool 
7:00PM ERR error="required flag(s) \"email\" not set" success=false

### Version

ipatool version 2.1.4

### Relevant log output

```shell
7:00PM ERR error="required flag(s) \"email\" not set" success=false
```

**Comment by Tarun5139** [comment]
Could please let me know correct command for it ?
<img width="1691" alt="Screenshot 2024-09-26 at 6 55 29 PM" src="https://github.com/user-attachments/assets/175387eb-4bb7-4eb2-8bc5-bc50bf7c2903">

**Comment by Tarun5139** [comment]
Its urgent can you ?

**Comment by majd** [comment]
Please share more information about the issue you are having, such as the command that you are attempting to use to log in via ipatool.

**Comment by akbaig** [comment]
`ipatool auth login` is the command

**Comment by akbaig** [comment]
> ### What happened?
> Hello I am not able to login in ipa tool 7:00PM ERR error="required flag(s) "email" not set" success=false
> 
> ### Version
> ipatool version 2.1.4
> 
> ### Relevant log output
> ```shell
> 7:00PM ERR error="required flag(s) \"email\" not set" success=false
> ```

`ipatool auth login --email <YOUR_EMAIL>` should work!

**Comment by majd** [comment]
The issue should be fixed in #316.
